.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\kashi\\.windows-build-tools\\python27\\python.exe"
```

Comment:
Ignore `.vscode` configurations. 
@Kashish121 Add `.vscode` to `.gitignore`.

Reply:
@Kashish121 Delete `settings.json`. The local settings shouldn't be pushed to the GitHub.